thirdparty/mbedtls/LICENSE

```diff
@@ -200,3 +207,347 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+===============================================================================
+
+
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We shouldn't include the text of the GPL, this will freak people out. The upstream Apache-licensed source tarball usually has the edited version of the LICENSE with only Apache (basically the file we already had).